apps/visualization/rest_filters.py

```diff
@@ -23,6 +25,36 @@ def filter_topic(self, qs, name, value):
         return qs.filter(topic__in=topics)
 
 
+class SourceListAggFilter(filters.FilterSet):
+    emergency = filters.CharFilter(field_name='emergency')
+    iso3 = filters.CharFilter(field_name='iso3', method='filter_iso3')
+    source_date_from = filters.DateFilter(field_name="source_date", lookup_expr='gte')
+    source_date_to = filters.DateFilter(field_name="source_date", lookup_expr='lte')
+
+    class Meta:
+        model = SourceListAgg
+        fields = ()
+
+    def filter_iso3(self, qs, name, value):
+        if not value:
+            return qs
+        return qs.filter(iso3__icontains=value)
+
+    @property
+    def qs(self):
+        qs = super().qs
+        date_from = self.request.GET.get('source_date_from')
+        date_to = self.request.GET.get('source_date_to')
```

Comment:
Best to use `self.data` instead. Raw value is sent to self.data when using django_filters
`self.request.GET`/`self.request.query_params` depends on the endpoint type and framework.